app/javascript/dashboard/components/widgets/conversation/bubble/Text.vue

```diff
@@ -60,11 +60,14 @@ export default {
 .text-content {
   overflow: auto;
 
-  &::v-deep {
-    ul,
-    ol {
-      margin-left: var(--space-normal);
-    }
+  ol,
+  ul {
+    padding-inline-start: revert;
```

Comment:
Can you add `padding-left: var(--space-two)` and remove this line. So both will align properly as in the screenshot.

![image](https://user-images.githubusercontent.com/64252451/135554600-18a93b1d-a02d-4bee-8eb4-3e1bac46617c.png)

Reply:
@iamsivin Thank you for your comment. I doubt whether the same issue will arise again if the list order crossed 3-4 digits (not sure how often it cross that range though), that's why I tried resetting the padding for the ordered list. 

As per my testing 100+ is fine with this 2rem. Please find the screenshot. 

<img width="300" alt="Screenshot 2021-10-01 at 10 08 05 AM" src="https://user-images.githubusercontent.com/23727388/135566064-ccbcfa19-5f6b-4a4a-af6f-f6857eeaa886.png">

Reply:
@SanjuSudheerM  I think it will only happen in rare cases.